lib/checks/001-deprecations.js

```diff
@@ -21,6 +21,11 @@ checkDeprecations = function checkDeprecations(theme, themePath) {
             ruleCode: 'GS001-DEPR-IMG'
         },
         {
+            helperRegEx: /{{\s*?cover\b[\w\s='"]*?}}/g,
```

Comment:
This is just a data attribute ( there won't won't be any valid text like absolute='true') therefore we can follow the format `\s*?` instead of `\b[\w\s='"]*?`